edxval/migrations/0006_auto_20170822_0722.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-08-22 11:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.utils.timezone
+import edxval.models
+import model_utils.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('edxval', '0005_videoimage'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='VideoTranscript',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created')),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified')),
+                ('video_id', models.CharField(help_text=b'It can be an edx_video_id or an external video id', max_length=255)),
+                ('transcript', edxval.models.CustomizableFileField(blank=True, null=True)),
+                ('language_code', models.CharField(db_index=True, max_length=8)),
+                ('provider', models.CharField(choices=[(b'Custom', b'Custom'), (b'3PlayMedia', b'3PlayMedia'), (b'Cielo24', b'Cielo24')], default=b'Custom', max_length=30)),
+                ('file_format', models.CharField(choices=[(b'srt', b'SubRip'), (b'sjson', b'SRT JSON')], db_index=True, max_length=20)),
+            ],
+        ),
+        migrations.RemoveField(
+            model_name='subtitle',
+            name='video',
+        ),
+        migrations.DeleteModel(
+            name='Subtitle',
```

Comment:
We've to roll this out carefully. I think the way is to remove usages first and then in 2nd commit we add migrations. I could be wrong too.